app/assets/stylesheets/pricing_table.scss

```diff
@@ -9,7 +9,7 @@
 }
 
 .pricing-table__option {
-  background-color: $pages-background;
+  background-color: #e2e1de;
```

Comment:
Er, are we sure we're not going to use these colors again? Otherwise we should name em in `unstructured/_settings`